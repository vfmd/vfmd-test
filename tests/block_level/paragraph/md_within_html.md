This is a paragraph of text.

<div>
Markdown-processing *is not* done here.

Markdown-processing *is* done here.

Markdown-processing *is* done here.
</div>

This is the last paragraph.
