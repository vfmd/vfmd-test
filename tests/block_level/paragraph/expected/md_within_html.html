<p>This is a paragraph of text.</p>

<div>
Markdown-processing *is not* done here.

<p>Markdown-processing <em>is</em> done here.</p>

Markdown-processing <em>is</em> done here.
</div>

<p>This is the last paragraph.</p>
